packages/map-template/src/components/Wayfinding/Wayfinding.scss

```diff
@@ -207,4 +199,8 @@
         background: var(--color-gray-10);
         border: 1px solid var(--color-gray-30);
     }
+
+    mi-search{
+        width: 90%;
```

Comment:
Why exactly 90%? Would it be more exact to use a `calc(100% - "whatever it needs to not cover ...px");` ? 😄

Reply:
Good idea, updated it ✅